openpype/settings/entities/color_entity.py

```diff
@@ -64,6 +66,6 @@ def convert_to_valid_type(self, value):
             new_value.append(item)
 
         # Make sure
-        if not self.use_alpha:
+        if hasattr(self, "use_alpha") and not self.use_alpha:
```

Comment:
```suggestion
        if not self.use_alpha:
```

Reply:
I'm sorry but I won't approve this. Just move initialization of `use_alpha` attribute before `self.value_on_not_set` and this is not needed.

Reply:
Omg, stupid GitHub showing old code...